torch/_inductor/runtime/hints.py

```diff
@@ -20,7 +20,8 @@ class ReductionHint(Enum):
     INNER = 0
     OUTER = 1
     OUTER_TINY = 2
-    DEFAULT = 3
+    DEFERRED_SPLIT = 3
```

Comment:
It's a little weird for this to show up at runtime, since it's a compile time only construct.